src/app/frontend/logs/component.js

```diff
@@ -123,12 +131,15 @@ export class LogsController {
    * @private
    */
   registerIntervalFunction_() {
```

Comment:
After starting it, interval function will be running in the background forever. I'd start/stop whole interval function on button click. It returns interval id and `interval.cancel(id)` can be used to stop it.